### PR TITLE
[Debt - Issue #158] added reset store action

### PR DESCRIPTION
**Pull request by rfreitas**
Issue here: https://github.com/onfido/onfido-sdk-ui/issues/158

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://206-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-206-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://206-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-206-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://206-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-206-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://206-pr-onfido-sdk-ui-onfido.surge.sh

File: src/core/store/actions/index.js [hunk 1]
```diff
@@ -2,6 +2,9 @@ import { bindActionCreators } from 'redux'
 import store from '../store'
 import * as globals from './globals'
 import * as captures from './captures'
+import * as constants from '../../constants'
```
Comment (hjheath) [inline src/core/store/actions/index.js]: Can we do an explicit import here (and in `reducers/index.js`)?

File: deploy.sh [hunk 2]
```diff
@@ -38,12 +38,13 @@ for DEPLOY_SUBDOMAIN_UNFORMATTED in "${DEPLOY_SUBDOMAIN_UNFORMATTED_LIST[@]}"
...
   echo $DEPLOY_SUBDOMAIN_UNFORMATTED
 
-  # replaces "/" or "." with "-"
+  # replaces non alphanumeric symbols with "-"
   # sed -r is only supported in linux, ref http://stackoverflow.com/a/2871217/689223
   # Domain names follow the RFC1123 spec [a-Z] [0-9] [-]
   # The length is limited to 253 characters
   # https://en.wikipedia.org/wiki/Domain_Name_System#Domain_name_syntax
-  DEPLOY_SUBDOMAIN=`echo "$DEPLOY_SUBDOMAIN_UNFORMATTED" | sed -r 's/[\/|\.]+/\-/g'`
+  DEPLOY_SUBDOMAIN=`echo "$DEPLOY_SUBDOMAIN_UNFORMATTED" | sed -r 's/[^A-Za-z0-9]+/\-/g'`
```
Comment (hjheath) [inline deploy.sh]: Nice! (although it's still bash....)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-206-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://206-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**